common/TracyQueue.hpp

```diff
@@ -124,6 +125,10 @@ struct QueueZoneValidation
     uint32_t id;
 };
 
+struct QueueZoneColor {
+  uint32_t value;
```

Comment:
This should be 3 bytes (alpha is ignored).

Reply:
ah - I didn't see Int24 being used with color [here](https://github.com/wolfpld/tracy/blob/4eb489508def47ca4c64af099050a3607245e0c6/server%2FTracyEvent.hpp#L167) so thought there may have been a reason - changed!

Reply:
Source locations are low volume, so it doesn't matter that much if everything is optimally packed. Zone colors can be sent for each zone.

Reply:
It looks like Int24 isn't accessible from common/ (it's defined in server/TracyEvent.hpp) - should I just use uint8_t[3] and cast it around?

Reply:
nevermind, I see SourceLocation has a split r g b - I'll do the same.